Source/Parallelization/WarpXComm_K.H

```diff
@@ -69,560 +69,201 @@ void warpx_interp (int j, int k, int l,
 }
 
 AMREX_GPU_DEVICE AMREX_FORCE_INLINE
```

Comment:
Could add a quick docstring to this function (as well as the `warpx_interp` function above that has the same name but a different signature), in order to explain the difference between these two functions. Also: in the doctring, could you specify whether `arr_fine_stag` applies to `arr_fine` or `arr_aux` (same comment for `arr_stag` in the above function `warpx_interp` of the same name).